### PR TITLE
(stoa0045a)

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated the cts_.xml file and bibliographic metadata.
 Combined two work files into one work file and updated the data.

### CODE REVIEW

**Comment by lcerrato** [comment]
@AlisonBabeu 
Always be sure to add your information and a description of what you've done to the change log of the file header when making major changes.

**Comment by AlisonBabeu** [comment]
so @lcerrato it looks like your suggestion worked so if you're ok with it I'll close this one.

**Comment by lcerrato** [comment]
@AlisonBabeu Sure, go for it!

File: data/stoa0045a/__cts__.xml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 <ti:textgroup xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:stoa0045a">
-  <ti:groupname xml:lang="eng">Catholic Church Pope</ti:groupname>
+  <ti:groupname xml:lang="eng">Catholic Church. Pope</ti:groupname>
```
Comment (lcerrato) [inline data/stoa0045a/__cts__.xml:2]: This is `Catholic Church. Pope` here and `Catholic Church Pope` below.

**Comment by lcerrato** [comment]
@AlisonBabeu  Remember to add a changelog see https://github.com/OpenGreekAndLatin/csel-dev/pull/370 for an example.

**Review by lcerrato — CHANGES_REQUESTED** [review]
Add change log

File: data/stoa0045a/__cts__.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/stoa0045a/__cts__.xml:2]: I will fix that.

**Review by lcerrato — APPROVED** [review]

**Pull request merged**